src/hooks/__tests__/useLoadDiscipline.spec.ts

```diff
@@ -93,13 +99,13 @@ describe('loadDiscipline', () => {
       expect(getFromEndpoint).toHaveBeenCalledWith('disciplines_by_level/', undefined)
     })
 
-    it('has no leaf parent without an api key', async () => {
+    it('has parent without an api key', async () => {
       await loadDisciplines({ ...parent, isLeaf: false })
       expect(getFromEndpoint).toHaveBeenCalledWith('disciplines_by_level/1234', undefined)
     })
 
-    it('has leaf parent', async () => {
-      await loadDisciplines(parent)
+    it('has training sets as children', async () => {
```

Comment:
please do not reproduce the training-sets naming from the api, as we want to get rid of it. 
`has training sets as children` is same as `isLeaf`

Reply:
It is not the same. `isLeaf` should mean that there are no more disciplines as children but documents, until now it just meant that we need another endpoint (training sets instead of disciplines) for which the naming is just wrong and really confusing.

Reply:
With the previous logic the second last level disciplines had `isLeaf` set to true while the last level had `isLeaf` set to false. Therefore it was not possible to decide which wording (`Wort` vs `Modul`) to use only depending on the discipline.

Reply:
No, isLeaf means it is the last discipline in the tree, which is a tree of disciplines.
Disciplines are groups, disciplines and training-sets. Documents are no disciplines, therefore words/documents are not part of the tree.

Distinction between Word and Module are fine (and currently working) with isLeaf-property, as if a discipline isLeaf it has words as children. In the ExersiceOverviewScreen we can always just use "words" as unit.

Reply:
Yes, exactly what I am saying. With the previous implementation training sets where NOT leafs while the discipline having training sets as children were leafs